src/internal/connector/data_collections.go

```diff
@@ -47,7 +47,7 @@ func (gc *GraphConnector) ProduceBackupCollections(
 		diagnostics.Index("service", sels.Service.String()))
 	defer end()
 
-	err := verifyBackupInputs(sels, gc.GetSiteIDs())
+	err := verifyBackupInputs(sels, gc.IDNameLookup.IDs())
```

Comment:
is this still all site IDs?

Reply:
Yep.  id-name guarantees should be complete by this stage.